swagger.yaml

```diff
@@ -158,6 +158,38 @@ paths:
       responses:
         204:
           description: OK
+  /asset:
+    get:
+      tags:
+        - Asset
+      description: Get an asset by id.
+      responses:
+        200:
+          description: OK
+        404:
```

Comment:
It seems that the express app does not return a 404 when the returned asset is not there (aka, empty/null), remove this error state to reflect the current state of the API.